packages/@sanity/vision/src/components/VisionGui.tsx

```diff
@@ -704,12 +698,6 @@ export class VisionGui extends PureComponent<VisionGuiProps, VisionGuiState> {
     try {
       el.select()
       document.execCommand('copy')
-      this.props.toast.push({
-        closable: true,
-        title: 'Copied to clipboard',
-        status: 'info',
-        id: 'vision-copy',
-      })
```

Comment:
I don't know if I agree with this one 🤔 
It's pretty common behaviour (example of linear)

Uploading Screen Recording 2025-02-21 at 10.11.59.mov…